### PR TITLE
Fix failing unit tests

### PR DESCRIPTION
**Pull request by mjwitte**
Prompted by #4808 found a fix for PurchasedAirManager unit tests to run successfully under both release and debug builds.

### CODE REVIEW

**Comment by Myoldmopar** [comment]
!Nice!
 I hope ci likes it too!

**Comment by mjwitte** [comment]
I give it a 50/50 chance.

**Comment by mjwitte** [comment]
@Myoldmopar @jasondegraw So, these test results show that ZoneTempPredictorCorrector.CorrectZoneHumRatTest is crashing (along with the ISC).  Not sure where to go with these, since they all run successfully on my windows machine with release and debug.

**Comment by mjwitte** [comment]
@Myoldmopar Will be adding another commit shortly, just waiting to test a build before pushing it up.

File: tst/EnergyPlus/unit/ICSCollector.unit.cc [hunk 1]
```diff
@@ -19,8 +20,9 @@ using namespace EnergyPlus::ConvectionCoefficients;
 using namespace EnergyPlus::DataSurfaces;
 using namespace EnergyPlus::DataHeatBalance;
 using namespace EnergyPlus::DataHeatBalSurface;
+using namespace EnergyPlus::Psychrometrics;
 
-TEST( ISCSolarCollectorTest, BugTest ) {
+TEST( ISCSolarCollectorTest, CalcPassiveExteriorBaffleGapTest ) {
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/ICSCollector.unit.cc]: @jasondegraw This includes a name change for the ICS unit test, per your request.

**Comment by jasondegraw** [comment]
@mjwitte Is this ready to get merged in? If so I'll merge it sometime this morning.

**Comment by mjwitte** [comment]
@jasondegraw It can be.  It's step in the right direction, but doesn't solve all the issues yet.

**Comment by mjwitte** [comment]
@jasondegraw @Nigusse Pushed up a few more lines of code trying to get ICSSolarCollectorTest to run clean.

**Comment by Nigusse** [comment]
@mjwitte Thanks. was this an initialization problem only or you did other changes as well?

**Comment by mjwitte** [comment]
It was all unitialized global or other nonlocal variables. 



<div>-------- Original message --------</div><div>From: Bereket Nigusse <notifications@github.com> </div><div>Date:03/24/2015  3:50 PM  (GMT-06:00) </div><div>To: NREL/EnergyPlus <EnergyPlus@noreply.github.com> </div><div>Cc: "Michael J. Witte" <mjwitte@gard.com> </div><div>Subject: Re: [EnergyPlus] Hotfix purchased air unit test and
  ICSSolarCollectorTest  (#4809) </div><div>
</div>@mjwitte Thanks. was this an initialization problem only or you did other changes as well?

**Pull request merged**